org.kde.granatier.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "org.kde.granatier",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "granatier",
+    "rename-icon": "granatier",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11", 
```

Comment:
```suggestion
        "--socket=fallback-x11", 
```

Reply:
@barthalion I'm curious what's the difference between `--socket=x11` and `--socket=fallback-x11`?
Because the other KDE apps use `--socket=x11` rather than `--socket=fallback-x11`, if it's important then we should change it all.

Reply:
`fallback-x11` makes X11 socket available only if there's no Wayland socket present. The old `x11` exposes the socket unconditionally.

Reply:
Thanks!